### PR TITLE
Update Command-Line-Options.md

### PR DESCRIPTION
**Pull request by Zachinquarantine**
replace "berlin" with "london", and "yolov3" with "calaveras"

### CODE REVIEW

File: docs/_interface/Command-Line-Options.md [hunk 1]
```diff
@@ -43,17 +43,17 @@ COMMANDS:
 
 ETHEREUM OPTIONS:
   --config value                      TOML configuration file
-  --datadir value                     Data directory for the databases and keystore (default: "~/.ethereum")
+  --datadir value                     Data directory for the databases and keystore (default: "C:\\Users\\mihal\\AppData\\Local\\Ethereum")
```
Comment (karalabe) [inline docs/_interface/Command-Line-Options.md:46]: Please revert this line.

File: docs/_interface/Command-Line-Options.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ ETHASH OPTIONS:
   --ethash.cachesinmem value          Number of recent ethash caches to keep in memory (16MB each) (default: 2)
   --ethash.cachesondisk value         Number of recent ethash caches to keep on disk (16MB each) (default: 3)
   --ethash.cacheslockmmap             Lock memory maps of recent ethash caches
-  --ethash.dagdir value               Directory to store the ethash mining DAGs (default: "~/.ethash")
+  --ethash.dagdir value               Directory to store the ethash mining DAGs (default: "C:\\Users\\mihal\\AppData\\Local\\Ethash")
```
Comment (karalabe) [inline docs/_interface/Command-Line-Options.md:87]: And this one too

File: docs/_interface/Command-Line-Options.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ ETHASH OPTIONS:
   --ethash.cachesinmem value          Number of recent ethash caches to keep in memory (16MB each) (default: 2)
   --ethash.cachesondisk value         Number of recent ethash caches to keep on disk (16MB each) (default: 3)
   --ethash.cacheslockmmap             Lock memory maps of recent ethash caches
-  --ethash.dagdir value               Directory to store the ethash mining DAGs (default: "~/.ethash")
+  --ethash.dagdir value               Directory to store the ethash mining DAGs (default: "~/.ethash)
```
Comment (holiman) [inline docs/_interface/Command-Line-Options.md:87]:
```suggestion
  --ethash.dagdir value               Directory to store the ethash mining DAGs (default: "~/.ethash")
```

**Review by holiman — APPROVED** [review]

**Comment by holiman** [comment]
Thanks!

**Pull request merged**